modMcf/src/org/aion/mcf/config/CfgApiZmq.java

```diff
@@ -38,19 +38,23 @@
 
 public class CfgApiZmq {
 
+    public static final String ZMQ_KEY_DIR = "zmq_keystore";
+
     CfgApiZmq() {
         this.active = true;
         this.ip = "127.0.0.1";
         this.port = 8547;
         this.filtersEnabled = true;
         this.blockSummaryCacheEnabled = false;
+        this.secureConnectEnabled = false;
```

Comment:
optional: might be safer to default it to true (after all, it's true by default in config.xml)

Reply:
the will notice in the document and wiki later.